nntrainer/tensor/cl_operations/blas_kernel_interface.h

```diff
@@ -70,5 +70,14 @@ void multiplyCl(Tensor &input, float const &value);
  */
 void add_i_cl(Tensor const &input, Tensor &result);
 
+/**
+ * @brief Process data and dimensions for transpose operation
+ * @param[in] direction string
+ * @param[in] input Tensor
+ * @param[in] result Tensor
+ */
+void Transpose_i_cl(const std::string &direction, Tensor const &in,
```

Comment:
I think it might be better to follow the naming convention.
What about changing the first letter as lowercase?
In addition, why do you name this with `_i` ? It seems not the inplace operation though.
```suggestion
void transpose_cl(const std::string &direction, Tensor const &in,
```

Reply:
Updated it, thanks! Please review.